Tests/SwiftStdlibTests/StringExtensionsTests.swift

```diff
@@ -77,7 +77,12 @@ final class StringExtensionsTests: XCTestCase {
     func testIsEmail() {
         XCTAssert("john@appleseed.com".isEmail)
         XCTAssertFalse("john@appleseedcom".isEmail)
-        XCTAssertFalse("johnappleseed.com".isEmail)
+		XCTAssertFalse("johnappleseed.com".isEmail)
```

Comment:
You're indenting with tabs here, while it looks like the rest uses spaces. Should we have a defined standard?

Reply:
💯true, will update this